mbed-events/README.md

```diff
@@ -0,0 +1,153 @@
+## The mbed-events library ##
```

Comment:
This should move to the handbook and get a light dusting to reflect the integration?